### DISCUSSION TITLE
Blog Post [3]

### DISCUSSION THREAD

**Discussion by gordi76** [NicoReimer/Breakpoint#9] [Weekly-Blog]
<h1>Use-Case-Realization Specification</h1>

Diese Woche haben wir einige Sequenzdiagramme und UML-Aktivitätsdiagramme erstellt. Die neue Version des [SRS](https://github.com/NicoReimer/Breakpoint/blob/main/SRS-Wiki-02.pdf) wurde hochgeladen.
Zusätzlich haben wir zwei UCRS erstellt: [UCRS User Authentication](https://github.com/NicoReimer/Breakpoint/blob/main/UCRS-UserAuthentication.pdf) und [UCRS Wiki Page Creation](https://github.com/NicoReimer/Breakpoint/blob/main/UCRS-WikiPageCreation.pdf).

**Comment by Hpixpoke** [comment]
Moin,

eure UCRS' sehen sehr gut aus - strukturiert und scharf formuliert! Die Sequenzdiagramme sehen ebenfalls sehr gut aus; evtl. kann man, wie auf S. 11, mehrere Pfeile zusammenfassen? Die Aktivitätsdiagramme sind einfach zu verstehen und decken alle Fälle ab.

Gut gemacht!

VG
Hagen und Team Büroplatzverwaltung

**Comment by GermanJesus-lul** [comment]
Hey,

eure SRS und UCRS' sehen schonmal gut aus.
Habt ihr euch schon Gedanken um die Inhaltseingabe beim Erstellen einer Wiki-Seite gemacht? Genauer, ob Styling-Optionen wie fett, Titel, Abschnitte etc. möglich sind und wie der Nutzer diese angibt.

Mit freundlichen Grüßen,
Julius und das Rezeptbuch Team
